### PR TITLE
feat: added grid component

### PR DESCRIPTION
**Pull request by HeyNitin**
Added grid component

image

### CODE REVIEW

File: Components/Grids/grids.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <body>
+    <div class="grid-two">
+      <div>First column</div>
+      <div>Second column</div>
+    </div>
+
+    <div class="grid-three">
+      <div>First column</div>
+      <div>Second column</div>
+      <div>Second column</div>
```
Comment (TejasShekar) [inline Components/Grids/grids.html:19]: *Third Column

**Comment by ajit-sharmaGH** [comment]
Looks fine to merge 😎

**Review by TejasShekar — COMMENTED** [review]
Looks good but from the image provided, seems like there's no gap in grid-two

File: Components/Grids/grids.html (hunk 1 above)
Comment (HeyNitin) [reply Components/Grids/grids.html:19]: Hey, yes, I've fixed that. Thanks for pointing it out Tejas.

**Pull request merged**